### ISSUE TITLE
Custom 404 Page

### ISSUE THREAD

**Issue by venom1979** [caffeinated/themes#99]
How can I set a custom 404 page using the current theme? Is this possible?

**Comment by kaidesu** [comment]
totally :+1: nothing special should be needed here - Laravel looks for custom error pages within an `errors` folder, so you'll want to create `errors/404.blade.php` to create a custom 404 page.

https://laravel.com/docs/5.8/errors#custom-http-error-pages

**Comment by venom1979** [comment]
Well considering that all of the theme files are hosted in their own folders (themes/ThemeName), and the errors folder must be included in the resources/views folder, that's not really the case. Meaning, if I switch themes, it will still default to the resources/views/errors/404.blade.php file, which does not change when I switch the current theme. I tried creating an errors folder within the theme folder and it still defaults to the resources/views/errors folder rather than the current theme. 

I was able to get around this by using @include('errors.404) in the resources/errors/404.blade.php file, but this is more like a hack than anything else.
